src/help.c

```diff
@@ -242,6 +242,7 @@ static void help_draw_chat(ToxWindow *self)
     wprintw(win, "  /sdev <type> <id>          : Change active device\n");
     wprintw(win, "  /mute <type>               : Mute active device if in call\n");
     wprintw(win, "  /sense <n>                 : VAD sensitivity threshold\n");
+    wprintw(win, "  /bitrate <n>               : Set the audio encoding bitrate\n");
```

Comment:
You also need to update the appropriate height parameters in help_onKey(). You'll probably want to do a bit of testing to make sure it looks right.